.github/workflows/kind.yml

```diff
@@ -91,13 +92,16 @@ jobs:
       run: |
         make e2e-test-uninstall-coverage
 
+    - name: Set up Go 1.22 just for coverage reports
+      uses: actions/setup-go@v3
+      with:  
+        go-version: '1.22'
+
     - name: Test Coverage and Report Generation
       if: ${{ matrix.kind == 'latest' }}
       run: |
         make test-coverage | tee report_unit.json
         make coverage-verify
```

Comment:
For other PRs to 2.8/2.9 we've just removed the check, but this way is fine too. 🙂 
```suggestion 
```